immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,880 @@
+{
```

Comment:
<p>We could consider adding a description here about about how to view the progress of the HP tuning job. Since the output of the previous cell does not update it's hard to know when the job updates. Perhaps, can say:</p><p> </p><p>Go to the Training dashboard and view the progression of the HP tuning job under "Hyperparameter Tuning Jobs"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
Done!